sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -167,6 +167,9 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog {
       case _: ShowTableExtended =>
         throw new AnalysisException("SHOW TABLE EXTENDED is not supported for v2 tables.")
 
+      case c: Command =>
+        c.plansToCheckAnalysis.foreach(checkAnalysis)
```

Comment:
This seems hacky? But we cannot make the analyzed plan as `children` for `CreateViewCommaned`. The reason is that the `View` will be optimized away (in the optimizer), and the verification that checks if a permanent view references temp views will fail.